src/components/AddressBook/AddressBookTableRow.tsx

```diff
@@ -34,32 +36,35 @@ export const AddressBookTableRow: FunctionComponent<AddressBookTableRowProps> =
   source,
 }) => {
   return (
-    <tr onClick={onAddressSelect} data-testid="table-row">
-      <th>{name}</th>
-      <td>{address}</td>
-      <td>{!isLocal && source}&nbsp;</td>
-      <td>{network && <AddressBookEtherscanLink address={address} network={network} />}</td>
-    </tr>
+    <div
+      className={`flex flex-col text-cloud-900 text-sm p-4 md:flex-row ${
+        (id + 1) % 2 !== 0 ? "bg-cerulean-50" : "bg-white"
+      }`}
```

Comment:
```suggestion
      className={`flex flex-col text-cloud-900 text-sm p-4 md:flex-row even:bg-cerulean-50`}
```

u need to enable even classes in config then it will work -> https://tailwindcss.com/docs/hover-focus-and-other-states#even-child

thereafter, remove  passing of `id` props

Reply:
done.

Reply:
oh u didnt have to enable `even` via tailwind config? works out of box?

Reply:
need. i just had to add even/odd in the extended backgroundColor properties inside tailwind.js.